### PR TITLE
Edit data-sync for style (partial)

### PR DESCRIPTION
**Pull request by finp**

### CODE REVIEW

File: modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc [hunk 1]
```diff
@@ -61,7 +61,9 @@ exampleQuery(...) @onlineOnly {
 [#sync-client-offline-queue-listener]
 == Listening for Events
 
-In order to get notification about offline related events, the `offlineQueueListener` in the config object should be used. The following events are emitted:
+To handle all notifications about offline related events, use the *offlineQueueListener`* listener in the config object
```
Comment (psturc) [inline modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc]: small typo \*offlineQueueListener`\* -> \*offlineQueueListener\*

File: modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc [hunk 2]
```diff
@@ -19,20 +20,23 @@ For this use case Push Notifications are recommended.
...
-To set up a client to use subscriptions simply provide a wsUrl string in the config object.
+To set up a client to use subscriptions:
 
+. Provide a `wsUrl` string in the config object as follows:
++
 [source,javascript]
 ----
 const config = {
-    wsUrl: "ws://your_websocket_url"
+    <wsUrl>: "ws://your_websocket_url"
```
Comment (psturc) [inline modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc]: We want user to replace the **value** of the object's property here, i.e. `ws://your_websocket_url`, not `wsUrl`. So would replacing:
`<wsUrl>: "ws://your_websocket_url"` for `wsUrl: "<websocket_url>"` make more sense?

If so, let's replace `<wsUrl>` for `<websocket_url>` in the following sentence as well

File: modules/ROOT/pages/_partials/sync/server-auth.adoc [hunk 3]
```diff
@@ -88,13 +88,13 @@ The `@hasRole` usage is as follows:
   * If the authenticated user has the role `admin` they will be authorized.
 * `@hasRole(role: [String])`
   * Example - `@hasRole(role: ["admin", "editor"])`
-  * If the authenticated user has **at least one of the roles** in the list, they will be authorized.
+  * If the authenticated user has at least one of the roles in the list, they will be authorized.
 
-**The default behaviour is to check client roles.** For example, `@hasRole(role: "admin"])` will check that user has a client role called `admin`. `@hasRole(role: "realm:admin"])` will check if that user has a realm role called `admin`
+The default behaviour is to check client roles. For example, `@hasRole(role: "admin"])` will check that user has a client role called `admin`. `@hasRole(role: "realm:admin"])` will check if that user has a realm role called `admin`
```
Comment (psturc) [inline modules/ROOT/pages/_partials/sync/server-auth.adoc]: `@hasRole(role: "admin"])` -> `@hasRole(role: "admin")`
`@hasRole(role: "realm:admin"])` -> `@hasRole(role: "realm:admin")`
This typo was also in the older version.

**Comment by psturc** [comment]
@finp just a few comments, otherwise LGTM

File: modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc (hunk 2 above)
Comment (finp) [reply modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc]: doh! yep, that's what i was trying to do

**Comment by finp** [comment]
@psturc fixed the issues you spotted.
@jhellar  you still reviewing?

**Pull request merged**